index.src.html

```diff
@@ -650,6 +655,17 @@ <h4 id="html-iframe-element-attributes">Attributes</h4>
   The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
   <code>referrerpolicy</code> content attribute, <a>limited to only known
   values</a>.
+
+  <h3 id="html-link-element">HTMLLinkElement</h3>
+  <pre class="idl">
+    partial interface HTMLLinkElement {
+      attribute DOMString referrerPolicy;
+    };
+  </pre>
+  <h4 id="html-link-element-attributes">Attributes</h4>
+  The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
+  <code>referrerpolicy</code> content attribute, <a>limited to only known
+  values</a>.
```

Comment:
well, fetch doesn't talk about how it fetches css, or favicons, or what happens in the bookmark system, or how prerendering would work...